.tekton/integration-tests/pipelines/lightspeed-operator-e2e-test-pipeline-416.yaml

```diff
@@ -347,6 +347,41 @@ spec:
                 value: "quay.io/openshift-lightspeed/ols-operator-artifacts:416$(params.commit)"
               - name: credentials-volume-name
                 value: ols-konflux-artifacts-bot-creds
+          - name: cleanup-resources
+            onError: continue
+            volumeMounts:
+              - name: credentials
+                mountPath: /credentials
+            env:
+              - name: KUBECONFIG
+                value: "/credentials/$(steps.get-kubeconfig.results.kubeconfig)"
+            image: registry.redhat.io/openshift4/ose-cli:latest
+            script: |
+              #!/bin/bash
+              set -x
+              echo "=========================================="
+              echo "Cleaning up OLS resources after artifact push"
+              echo "=========================================="
+
+              # Delete all OLSConfig CRs in openshift-lightspeed namespace
+              echo "Deleting OLSConfig CRs..."
+              oc delete olsconfig --all -n openshift-lightspeed --ignore-not-found=true --timeout=60s || true
```

Comment:
$ oc delete --help
...
    --wait=true:
	If true, wait for resources to be gone before returning. This waits for finalizers.